src/rageshake/submit-rageshake.ts

```diff
@@ -32,7 +32,7 @@ interface IOpts {
     userText?: string;
     sendLogs?: boolean;
     progressCallback?: (string) => void;
```

Comment:
```suggestion
    progressCallback?: (s: string) => void;
```

Could you change this here just so it doesn't get forgotten about please